generators/client/templates/angular/home/home.component.html.ejs

```diff
@@ -17,7 +17,7 @@
         <span class="hipster img-fluid rounded"></span>
     </div>
     <div class="col-md-9">
-        <h1 class="display-4" jhiTranslate="home.title">Welcome, Node Hipster Official Blueprint!</h1>
+        <h1 class="display-4" jhiTranslate="home.title">Welcome, Java Hipster!</h1>
```

Comment:
Does this make sense? There's no Java in a Node.js-generated project, is there?

Reply:
The reason that I change it, is that we uses the e2e client jhipster tests (I have not overrided the templates of jhipster e2e tests, only for the administration because I have removed the admin java features not supported, such as the logs, metrics), that have assertions for the home title. However, if it is selected the i18n, the home title is replaced with the node hipster customization: https://github.com/jhipster/generator-jhipster-nodejs/blob/df15c0e86884b447a2c3a51a7d31895878d0c966/generators/languages/templates/src/main/webapp/i18n/en/home.json#L3